client/src/components/teams/TeamDashboard.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <v-container>
+    <v-row class="mx-6" :fullscreen="mobile">
+      <!-- Team Name -->
+      <v-row class="my-2">
+        <v-col align="center">
+          <h1>{{ teamData.team_name }}</h1>
+        </v-col>
+      </v-row>
+      <v-divider></v-divider>
+
+      <!-- Total Kilometres -->
+      <v-row align="center" class="my-2">
+        <v-icon class="mdi mdi-run-fast ml-3" size="50px"></v-icon>
+        <v-col>
+          <h2 class="totalKms">{{ teamData.total_kilometres }} KM</h2>
+          <h3>TOTAL</h3>
+        </v-col>
+      </v-row>
+      <v-divider></v-divider>
+
+      <!-- Invite Code -->
+      <v-row align="center" class="my-2">
+        <v-col>
+          <h2>Invite Code</h2>
+          <p class="invite-code">{{ teamData.invite_code }}</p>
+        </v-col>
+        <v-col align="end" class="px-5">
+          <v-icon
+            @click="copyInviteCode"
+            class="mdi mdi-clipboard-multiple-outline"
+            size="32px"
```

Comment:
it would be nice if something small popped up to let the user know its been copied. Theres a few options for this
https://vuetifyjs.com/en/components/tooltips/.  
https://vuetifyjs.com/en/components/alerts/
https://vuetifyjs.com/en/components/snackbars/

I'd probably go with the tooltip option